package.json

```diff
@@ -105,8 +106,13 @@
     "react-is": "17.0.2",
     "react-router-dom": "5.3.0",
     "react-toastify": "8.2.0",
-    "styled-components": "5.3.5",
     "ts-jest": "27.1.4",
     "typescript": "4.6.3"
+  },
+  "dependencies": {
+    "@mui/material": "^5.8.1",
+    "@mui/styled-engine-sc": "^5.8.0",
+    "@mui/styles": "^5.8.0",
+    "styled-components": "^5.3.5"
```

Comment:
Why did you move the packages from `devDependencies` to regular dependencies? I don't know why, but all packages were marked as `devDependencies`. Maybe that's a Nx thing, but let's keep it for now.